packages/ui/src/components/DataStory/Workbench.tsx

```diff
@@ -24,28 +24,77 @@ const nodeTypes = {
   tableNodeComponent: TableNodeComponent,
 };
 
-export const Workbench = ({
```

Comment:
When considering names for DataStory, AllotmentDataStory etc, ofc this component could also be renamed if we come up with some better name. Im not sure what its primary purpose is as of now other than hiding some ReactFlowProvider etc.

Reply:
`FlowEditor`: For editing flowcharts within DataStory project. 
What do you think of this name? :eyes: